### PR TITLE
Update community_connectors.json

### PR DESCRIPTION
**Pull request by badosa**

### CODE REVIEW

**Review by badosa — COMMENTED** [review]
There's a missing comma after "}".

**Review by theoutlander — CHANGES_REQUESTED** [review]

File: community/community_connectors.json [hunk 1]
```diff
@@ -502,6 +502,16 @@
...
+  {
+      "name": "JSON-stat (and SDMX-JSON) Connector",
+      "url": "https://jsonstat.com/tableau/",
+      "author": "Xavier Badosa",
+      "github_username": "badosa",
+      "tags": ["v_2.3"],
+      "description": "A general connector to import data from most statistical offices in the World.",
+      "source_code": ""
+  }
+
```
[the remark this answers is not in the document]
Comment (badosa) [reply community/community_connectors.json]: Comma added.

**Comment by badosa** [comment]
Comma added

**Comment by theoutlander** [comment]
Thanks, @badosa!

**Pull request merged**